core/src/main/scala/cats/data/Ior.scala

```diff
@@ -203,6 +204,39 @@ private[data] sealed abstract class IorInstances extends IorInstances0 {
     new Bifunctor[Ior] {
       override def bimap[A, B, C, D](fab: A Ior B)(f: A => C, g: B => D): C Ior D = fab.bimap(f, g)
     }
+
+  implicit def parallelForIor[E]
+    (implicit E: Semigroup[E]): Parallel[Ior[E, ?], Ior[E, ?]] = new Parallel[Ior[E, ?], Ior[E, ?]]
+  {
+
+    def parallel: Ior[E, ?] ~> Ior[E, ?] = FunctionK.id
```

Comment:
these could be `val` do avoid the reallocations, right?

Reply:
👍